src/pages/set-password.tsx

```diff
@@ -33,6 +34,8 @@ export const SetPasswordPage: React.FC<SetPasswordProps> = ({
   const { doSetPassword, wallet, doFinishSignIn } = useWallet();
   const doChangeScreen = useChangeScreen();
   const { decodedAuthRequest } = useOnboardingState();
+  const analytics = useAnalytics();
+  void analytics.page('view', '/set-password');
```

Comment:
This really needs to be in a `useEffect`. Even if this does only render once, because of the memo, we shouldn't rely on it, as that could easily change later. 

```ts
useEffect(() => {
 void analytics.page('view', '/set-password');
}, []);
```

Reply:
Yes indeed, useEffect is better.  I will fix it !